src/o365/spfx/commands/project/project-rename.ts

```diff
@@ -0,0 +1,193 @@
+import commands from '../../commands';
+import Command, {
+  CommandOption, CommandAction, CommandError, CommandValidate
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import { BaseProjectCommand } from './base-project-command';
+import * as path from 'path';
+import * as fs from 'fs';
+const uuid = require('uuid');
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  newName: string;
+  generateNewId?: boolean;
+}
+
+class SpfxProjectRenameCommand extends BaseProjectCommand {
+  public constructor() {
```

Comment:
No need to implement the constructor given we're not doing anything special here and the parent constructor will be called automatically.

Reply:
Agreed. I've removed the constructor now.